modules/builds-configuration-file.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * builds/build-configuration.adoc
+
+[id="builds-configuration-file_{context}"]
+= Configuring build settings
+
+You can configure build settings by editing the
+`build.config.openshift.io/cluster` file.
+
+.Procedure
+
+* Edit the `build.config.openshift.io/cluster` file:
++
+----
+$ oc edit build.config.openshift.io/cluster
+----
++
+The following is an example `build.config.openshift.io/cluster` file:
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Build
+metadata:
+  annotations:
+    release.openshift.io/create-only: "true"
+  creationTimestamp: "2019-05-17T13:44:26Z"
+  generation: 2
+  name: cluster
+  resourceVersion: "107233"
+  selfLink: /apis/config.openshift.io/v1/builds/cluster
+  uid: e2e9cc14-78a9-11e9-b92b-06d6c7da38dc
+spec:
+  additionalTrustedCA:
+    name: myconfigmap
+  buildDefaults:
+    defaultProxy:
+      httpProxy: http://proxy.com
+      httpsProxy: https://proxy.com
+      noProxy: internal.com
+    env:
+    - name: envkey
+      value: envvalue
+    gitProxy:
+      httpProxy: http://gitproxy.com
+      httpsProxy: https://gitproxy.com
+      noProxy: internalgit.com
+    imageLabels:
+    - name: labelkey
+      value: labelvalue
+    resources:
+      limits:
+        cpu: 100m
+        memory: 50Mi
+      requests:
+        cpu: 10m
+        memory: 10Mi
+  buildOverrides:
+    imageLabels:
+    - name: labelkey
+      value: labelvalue
+    nodeSelector:
+      selectorkey: selectorvalue
+    tolerations:
+    - effect: NoSchedule
+      key: node-role.kubernetes.io/builds
+operator: Exists
```

Comment:
would be nice if we can annotate this w/ footnotes that give a little more detail on each field

Reply:
Added